### DISCUSSION TITLE
⭐ FABRIC APP FRONT-END — Alpha ⭐

### DISCUSSION THREAD

**Discussion by noamsiegel** [danielmiessler/Fabric#1132] [Show and tell]
# Fabric App — Alpha
Finally, the first front-end for Fabric (that I can find) has been released. Click the image below to see the installation instructions video and the progress so far.

It is currently in Alpha, and I want as much feedback as possible. If you find bugs or have feedback, please post it on the fabric app discussion board.

[![Install Fabric App](https://github.com/user-attachments/assets/65b191ef-20c7-45b9-9cbd-f9e4672f4b15)](https://github.com/noamsiegel/fabric-app)

**Comment by jamescardona11** [comment]
Awesome, I'll definitely try it, it looks great.

**Reply by noamsiegel** [reply]
Thanks!

**Comment by swizzcheeze** [comment]
Definitely will be trying this when I get home tonight
